core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -200,11 +209,11 @@ public static void set(String key, String value) {
    * @return the value for the given key
    */
   public static String get(String key) {
-    if (!PROPERTIES.containsKey(key)) {
+    String raw = PROPERTIES_MAP.get(key);
+    if (raw == null) {
```

Comment:
This was changed to `get`, then check for `null`. Should we change the rest of this class to that model? @apc999